src/manager.rs

```diff
@@ -68,10 +68,15 @@ impl<T: MutEventSubscriber> SubscriberOps for EventManager<T> {
 impl<S: MutEventSubscriber> EventManager<S> {
     /// Create a new `EventManger` object.
     pub fn new() -> Result<Self> {
+        Self::new_with_capacity(256)
+    }
+
+    /// Creates a new `EventManger` object with specified event capacity.
```

Comment:
Can you also add an `Arguments` section and explain what ready_events_capacity is? It would be helpful to know when to use this constructor and not the default one.

Reply:
Added an argument section, kindly let me know if I need to alter the sentences 😃

Reply:
Looks good. The formatting needs an update, can you add a blank line before and after `#Arguments`?

Reply:
done 👍